NoteFind/Models/Book.m

```diff
@@ -10,6 +10,8 @@
 @implementation Book
 
 - (id)initWithDictionary:(NSDictionary *)dictionary {
+    //getting unique ID
```

Comment:
I assume the `id` is required in the dictionary? It would be a good practice to have an `NSAssert(...)` here to cover flows where the dictionary doesn't contain a ID

https://developer.apple.com/documentation/foundation/nsassert